pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -28,13 +28,14 @@ let
   };
 
 in buildPythonApplication rec {
-  name = "qutebrowser-${version}";
-  version = "1.0.2";
+  name = "qutebrowser-${version}${fix_postfix}";
+  fix_postfix = "-1";
+  version = "1.0.3";
   namePrefix = "";
 
   src = fetchurl {
```

Comment:
Please use ```fetchFromGitHub``` instead of ```fetchurl```.

Reply:
@peterhoeg No, this is a release made by the maintainers of the project, it is not autogenerated by GitHub.